core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -826,6 +830,18 @@ private DefaultFunctionResolver to_days() {
         impl(nullMissingHandling(DateTimeFunction::exprToDays), LONG, DATETIME));
   }
 
+  /**
+   * TO_SECONDS(STRING/DATE/DATETIME/TIMESTAMP). return the day number of the given date.
+   */
```

Comment:
Does MySQL or other DBs support FSP in this function?

Reply:
FSP? I'm not sure if you mean Function Properties, but I'm not sure how this function could support it.